src/NSubstitute/NSubstitute.csproj

```diff
@@ -47,7 +30,7 @@
 
   <!-- ReferenceAssemblies so we can build netfx targets on non-windows enviroments -->
   <ItemGroup Condition="'$(TargetIsNetFx)' == 'true'">
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" />
   </ItemGroup>
```

Comment:
I think it makes sense to move the whole section to common files, as they are always identical.
If any project is not targeting `net`, then it will be simply ignored.

It will let you eliminate the `TargetIsNetFx` variable, as it will be used in a single place only.

Reply:
Makes sense. I moved it to the `props` file and removed from individual `csproj`s. They all should get it now.